lib/reviewlette.rb

```diff
@@ -5,6 +5,10 @@
 
 VERSION = '0.0.10'
 
+# Assume cards have following card title when estimated
+# (8) This is the card name'
+POINTS_REGEX = /\(([\d.]+)\)/
```

Comment:
I just copied the regex from trollolo to match the regex from there. But your right